src/components/FieldDateInput/DateInput.js

```diff
@@ -151,6 +157,12 @@ class DateInputComponent extends Component {
 
     const date = value && value.date instanceof Date ? moment(value.date) : initialMoment;
 
+    const isDayBlocked = timeSlots
+      ? day => {
+          return !timeSlots.find(timeSlot => isSameDay(day, moment(timeSlot.attributes.start)));
```

Comment:
Is the `timeSlot.attributes.start` always a start of a single day or could it be a range of days?
Either way, it could be good to have a short comment about response restrictions and rules? 
Another related response format question: is the returned day in UTC 00:00 timestamp or has the timezone difference added to it already?

Reply:
This might help: https://github.com/sharetribe/flex-template-web/blob/master/src/util/dates.js#L27